index.html

```diff
@@ -139,17 +139,17 @@
 are designed to enable the controller of a <a>DID</a> to prove control over
 it and to be implemented independently of any centralized registry,
 identity provider, or certificate authority. <a>DIDs</a> are URLs that relate
-a <a>DID subject</a> to means for trustable interactions with that subject.
-<a>DIDs</a> resolve to <a>DID documents</a> &mdash; simple documents that describe how
+a <a>DID subject</a> to means for trustable interactions with that subject by way
+of a <a>DID document</a>. <a>DID documents</a> are simple documents that describe how
 to use that specific <a>DID</a>. Each <a>DID document</a> may express cryptographic
 material, verification methods, and/or <a>service endpoints</a>. These provide
 a set of mechanisms which enable a <a>DID controller</a> to prove control of the
 <a>DID</a>. <a>Service endpoints</a> enable trusted interactions with the <a>DID
 subject</a>.
     </p>
     <p>
-This document specifies a common data model, format, and operations that
-all <a>DIDs</a> support.
+This document specifies a common data model, a URL format, and a set of operations for
+<a>DIDs</a> and related components.
```

Comment:
"DIDs and related components" doesn't make much sense to me; what are those "related components"? Syntax components like DID path or DID query? Or architectural components like DID registry and DID resolver?

Reply:
I meant like DID documents and DID methods. I can change it to that.